src/containers/About.jsx

```diff
@@ -1,183 +1,93 @@
-                ></Card.Img>
-                <Card.Text className="exec-about">text</Card.Text>
-              </Card.Body>
-            </Card>
-            <Card className="card vpf">
-              <Card.Header>
-                <h3 className="exec-name">Mayank Kanoria </h3>
-                <h4 className="class-year">SE 2023</h4>
-                <h4 className="exec-role">VP Finance</h4>
-              </Card.Header>
-              <Card.Body>
-                <Card.Img
-                  variant="top"
-                  src={mayank}
-                  className="exec-img"
-                ></Card.Img>
-                <Card.Text className="exec-about">text</Card.Text>
-              </Card.Body>
-            </Card>
-          </div>
-        </div>
-        <div className="old">
-          <h2>Graduated Execs</h2>
-          <div className="team">
-            <Card className="card">
-              <Card.Header>
-                <h3 className="exec-name">Bilal Akhtar </h3>
-                <h4 className="class-year">SE 2019</h4>
-
-                <h4 className="exec-role">B-Soc President Winter 2019</h4>
-              </Card.Header>
-              <Card.Body>
-                <Card.Img
-                  variant="top"
-                  src={bilal}
-                  className="exec-img"
-                ></Card.Img>
-                <Card.Text className="exec-about">text</Card.Text>
-              </Card.Body>
-            </Card>
-          </div>
-        </div>
-      </div>
+      <Container className="mt-5 mb-5">
+        <h1>What is SE Soc?</h1>
+        <p>
+          Software Engineering Society (SE Soc) is a council that runs community
+          initiatives within the SE program. We give you a better shot at the
+          jobs you want by hosting resume critiques and interview prep sessions.
+          We build a sense of community by organizing class dinners and social
+          events. We introduce you to upper-years and lower-years by matching
+          you with someone to chat over coffee. We highlight the cool stuff that
+          everyone in SE is up to by producing our podcast, Sessions by SE.
+        </p>
+        <h1>Meet The Team</h1>
+        <p>
+          B-Soc represents the odd years (SE2021, SE2023, etc.) and A-Soc
+          represents the even years (SE2020, SE2022, etc.). We’re a friendly
+          bunch, feel free to say hi if we ever bump into each other on campus!
+        </p>
+        <ProfilePhotoGroup
+          soc={"B-Soc"}
+          execs={[
+            {
+              name: "Roxane Fruytier",
+              src: profileImage("roxane"),
+              year: "2021",
+              position: "President",
+            },
+            {
+              name: "Elisa Luan",
+              src: profileImage("elisa"),
+              year: "2021",
+              position: "VP Events",
+            },
+            {
+              name: "Mayank Kanoria",
+              src: profileImage("spencer"),
```

Comment:
oops guess we're still waiting on Mayank. (Is it too late to ask Ariel for a cute place holder character to use when we don't have picture of someone, or they would rather not have a picture on the website?)

Reply:
That's a great idea, it's never too late! I can ask ^^